### ISSUE TITLE
Missing electron bin folder

### ISSUE THREAD

**Issue by matejdro** [zokugun/vscode-sync-settings#53]
When downloading settings, I'm getting this error:

![image](https://github.com/zokugun/vscode-sync-settings/assets/507922/70ef23f5-d0f4-4644-afe3-b15ecd30fca3)

`/usr/lib/electron25` exists, but there is no bin folder. This is on Arch Linux. Electron is normally installed from arch package and as far as I can tell, works fine. Not sure why plugin expects bin folder to be there.

**Comment by matejdro** [comment]
It's the default [code](https://archlinux.org/packages/extra/x86_64/code/) package, `Code - OSS`.

**Comment by daiyam** [comment]
Yeah, the cli path is modified to `"$pkgdir"/usr/bin/code-oss` (https://gitlab.archlinux.org/archlinux/packaging/packages/code/-/blob/main/PKGBUILD?ref_type=heads#L127, https://gitlab.archlinux.org/archlinux/packaging/packages/code/-/blob/main/clipath.patch?ref_type=heads)

While the extension uses the current process to guess the app executable (https://github.com/zokugun/vscode-sync-settings/blob/master/src/utils/restart-app.ts#L70)

And since the `Code - OSS` use the system Electron to launch, the detection would need to be updated to support that version.
